servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/OpenTelemetryFilterBuilder.java

```diff
@@ -36,6 +40,8 @@ abstract class OpenTelemetryFilterBuilder<T extends OpenTelemetryFilterBuilder<T
     List<String> capturedResponseHeaders = emptyList();
     boolean enableMetrics;
     OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
+    @Nullable
+    Function<HttpRequestMetaData, String> spanNameExtractor;
```

Comment:
does the default extractor change or can we, similar to the other variables, make this non-nullable and default it to the default extractor?